templates/site/js/node.js

```diff
@@ -124,7 +127,18 @@ document.addEventListener("DOMContentLoaded", function () {
                 node_menu_item.setAttribute("tabindex", `${index - initial_node_count}`)
                 node_dropdown_content.appendChild(h5_dropdown_item);
             }
+
+            /*Builds dictionary to change colour of menu items in Node page and add hash to Node page URL*/
+            let options_object = {};
+            options_object["button"] = document.getElementById(key);
```

Comment:
you don't need to search for the element with the `getElementById` function since you have the elements as variables already.

Reply:
Added variable to the options_object instead of the command to get the DOM element.

Also changed how "click" attribute was added to `node_menu_item` so that `stop_propagation()` could be used to stop the "click" event from bubbling up to the parent "a" element.  Without this the menu colours would not change and the node details would not update.